tests/Elasticsearch/Tests/ClientIntegrationTests.php

```diff
@@ -18,9 +19,32 @@
  */
 class ClientIntegrationTests extends \PHPUnit\Framework\TestCase
 {
+
+    /**
+     * @var $client Elasticsearch\Client
+     */
+    protected static $client;
+
+    /**
+     * initialize elasticsearch client
+     */
+    public static function setUpBeforeClass()
+    {
+        self::$client = Elasticsearch\ClientBuilder::create()->setHosts([$_SERVER['ES_TEST_HOST']])->build();
+    }
+
+    public function tearDown()
+    {
+        // remove all previously indices created by test or by the before setup
+        $param = [
+            'index' => '_all'
+        ];
+        self::$client->indices()->delete($param);
+    }
+
     public function testCustomQueryParams()
     {
-        $client = Elasticsearch\ClientBuilder::create()->setHosts([$_SERVER['ES_TEST_HOST']])->build();
+        self::$client = Elasticsearch\ClientBuilder::create()->setHosts([$_SERVER['ES_TEST_HOST']])->build();
```

Comment:
Is this change necessary? I think that it is better if the tests do not depend on static attribute.